### DISCUSSION TITLE
Running on Colab ; TesseractNotFound

### DISCUSSION THREAD

**Discussion by edric-raharjo** [deepdoctection/deepdoctection#231] [Announcements]
I am using `deepdoctetion` in google colab

`!pip install deepdoctection[tf]`
This is fine

Then as usual, I followed the tutorial
```
import deepdoctection as dd
from IPython.core.display import HTML
from matplotlib import pyplot as plt
analyzer = dd.get_dd_analyzer()
```

On the `analyzer = ` line, I encountered an error both when using the CPU and the T4 GPU

Here is the error message on the CPU runtime
[error_on_cpu_runtime.txt](https://github.com/deepdoctection/deepdoctection/files/12483383/error_on_cpu_runtime.txt)

**Comment by JaMe76** [comment]
You need to install Tesseract separately as this cannot be installed through pip.

I suggest checking the web for the install.
 
There is plenty of help and the installation is straight forward.

**Comment by rogueloop** [comment]
got the same issue got fixed @edric-raharjo ?

**Comment by edric-raharjo** [comment]
Sorry for the quite late reply, I have been busy doing assignments, I have fixed it!

Use these
```
!pip install deepdoctection[tf]

!pip install thop
import thop

!pip install pytesseract
import pytesseract
pytesseract.pytesseract.tesseract_cmd = r'/usr/local/bin/pytesseract'
!sudo apt install tesseract-ocr
```
Then just go ahead and follow the documentation (adjust on your needs, whether it is a png or pdf file)

Edit : I forgot to mention, `thop` hasn't been downloaded, so make sure to download it first, from the sources I read, it's for counting FLOPs when using GPU, but take these words with a lot of salt (I have 0 knowledge on how GPU computing works)

**Comment by temiwale88** [comment]
Thanks @edric-raharjo . For others checking this out, you might need to restart (not delete) runtime for edric's fix to take effect.
